spec/controllers/orgs/lti_configurations_controller_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Orgs::LtiConfigurationsController, type: :controller do
+  let(:organization) { classroom_org                                   }
+  let(:user)         { classroom_teacher                               }
```

Comment:
Nit: this looks a little janky
```suggestion
  let(:user)         { classroom_teacher }
```